hooks/useChat.tsx

```diff
@@ -75,16 +74,12 @@ const useChat = () => {
   };
 
   const isPrepared = () => {
-    if (!address) {
-      connectWallet();
-      return false;
-    }
-    return true;
+    return authenticated && ready && !!address;
   };
 
   const clearQuery = async () => {
     if (!address) return;
-    await fetchInitialMessages(address);
+    await fetchInitialMessages(`0x${address}`);
```

Comment:
Address from Privy should already start with `0x`.

Why are you adding a prefix?
- `0x0x...` right?